.travis.yml

```diff
@@ -4,6 +4,12 @@ android:
   components:
     - build-tools-26.0.2
 
+before_cache:
```

Comment:
What does this do, and why is it needed on travis? Would be great with a comment that explains this as it's a target for cargo-culting otherwise. :)